### ISSUE TITLE
Additional spacing needed with offscreen text - bug for no-CSS - P2 bug

### ISSUE THREAD

**Issue by sareh** [BBC-archive/psammead#3126]
**Describe the bug**
When there is a live label, and we check the page with CSS turned off, there should be a space after the 1st live (visible) and before the 2nd live (visually hidden) so you can read this correctly e.g. 'LIVE Live,' rather than 'LIVElive,' 

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 

**Expected behaviour**
There should be a space after the 1st live (visible) and before the 2nd live (visually hidden) so you can read this correctly e.g. 'LIVE Live,' rather than 'LIVElive,' 
 

Can be fixed by passing in text ` Live,` for text within the VisuallyHiddenText component.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by OlgaLyubin** [comment]
Checked every component in Storybook with `Live` on and CSS disabled, and found the above issue only in `Bulletin` component. It is being fixed in the following PR: https://github.com/bbc/psammead/pull/3160

**Comment by OlgaLyubin** [comment]
This issue has been fixed for Bulletins:

<img width="614" alt="Screenshot 2020-03-02 at 12 26 45" src="https://user-images.githubusercontent.com/17802955/75676441-1de05880-5c81-11ea-92c5-f4bd4897df87.png">
